mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/logging/impl/Log.java

```diff
@@ -144,4 +144,8 @@
 
 	@Message(id = 1240, value = "Procedures returning muliple documents are not supported. Procedure '%1$s' returned %2$d results")
 	HibernateException multipleDocumentReturnedByStoredProcedure(String storedProcedureName, int size);
+
+	@LogMessage(level = WARN)
+	@Message(id = 1241, value = "Try to define another index on the same key set %s: definition of index %s will be skipped!")
```

Comment:
I would change this to something like:

`Skipping creation of Index <name of the index>.  Index <name of the index> already defined for properties %s.`

By the way, does this only apply for unique constraints?
In that case I would be more precise, like, `Skipping creation of index %s. Unique constraint on properties %s already defined by index %s`

Reply:
We can't know in this phase if the index is derived from a unique constraint or not. In fact a unique constraint it is equivalent to a normal unique index defined on same fields :(.

Reply:
Anyway I like the first message you proposed.